src/main/java/sceaj/adyenmock/api/v1/model/payment/Amount.java

```diff
@@ -0,0 +1,14 @@
+package sceaj.adyenmock.api.v1.model.payment;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Builder
+@Data
+public class Amount {
+
+    private String currency;
+    private BigDecimal value;
```

Comment:
This was my fault for assuming and not reading the documentation. For this moment  I defined it as long since in the documentation most currencies will have 2 decimals precision so having a payment in the million amount is possible.